polymer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "entrypoint": "app/1.0/index.html",
+  "shell": "app/elements/pw-shell.html",
+  "fragments": [
+    "app/elements/pw-shell.html",
```

Comment:
Talked offline: then it puts it in a `1.0` folder in the output; if i add it in the fragments, then i can get all my bundled elements from the same place. I'm holding the tool weirdly, look away :D